generator/Writer/Filesystem.php

```diff
@@ -41,8 +41,8 @@ public function __construct(string $root)
 
     public function clear()
     {
-        $this->flysystem->deleteDir('src');
-        $this->flysystem->createDir('src');
+        $this->flysystem->deleteDirectory('src');
+        $this->flysystem->deleteDirectory('src');
```

Comment:
Double deleting the directory seems wrong!? 🤔
I assume that the second one should be a `createDirectory()` call!?

Reply:
😮‍💨  Yep great catch - must have goofed up my paste when refactoring to the new method names.

Reply:
Just updated this - all fixed now!